tests/test_data_processor.py

```
@@ -0,0 +1,14 @@
+import pytest
+import pandas as pd
+import numpy as np
```

Ruff

`numpy` imported but unused

Remove unused import: `numpy`

(F401)